quantum/quantum_keycodes.h

```diff
@@ -446,25 +446,35 @@ enum quantum_keycodes {
     KC_LOCK,
 #endif
 
-#ifdef TERMINAL_ENABLE
-    TERM_ON,
-    TERM_OFF,
-#endif
-
-    EEPROM_RESET,
-
-    UNICODE_MODE_FORWARD,
-    UNICODE_MODE_REVERSE,
-
-    UNICODE_MODE_OSX,
-    UNICODE_MODE_LNX,
-    UNICODE_MODE_WIN,
-    UNICODE_MODE_BSD,
-    UNICODE_MODE_WINC,
-
-    // always leave at the end
-    SAFE_RANGE
-};
+    #ifdef TERMINAL_ENABLE
+        TERM_ON,
+        TERM_OFF,
+    #endif
+
+        EEPROM_RESET,
+
+        UNICODE_MODE_FORWARD,
+        UNICODE_MODE_REVERSE,
+
+        UNICODE_MODE_OSX,
+        UNICODE_MODE_LNX,
+        UNICODE_MODE_WIN,
+        UNICODE_MODE_BSD,
+        UNICODE_MODE_WINC,
+
+        HAPT_ON,
+        HAPT_OFF,
+        HAPT_TOGGLE,
```

Comment:
for Backlights we have TOGG, for RGB we have TOG, and now we have TOGGLE. LOL. I think we should try and stick with one spelling of toggle.

Reply:
I have changed the spelling to fit the 7 charater paradigm, so they are all not HPT_*

Reply:
Well, you can have both.  If you add a define below, .... that's how many keycodes are operating, some of mine included

Reply:
I think this may be fine for now without both, I've made a table in the docs to explain the keycodes anyways.